pkg/cmd/cli/serverstatus/server_status.go

```diff
@@ -40,7 +42,9 @@ type DefaultServerStatusGetter struct {
 func (g *DefaultServerStatusGetter) GetServerStatus(kbClient kbclient.Client) (*velerov1api.ServerStatusRequest, error) {
 	created := builder.ForServerStatusRequest(g.Namespace, "", "0").ObjectMeta(builder.WithGenerateName("velero-cli-")).Result()
 
-	if err := kbClient.Create(context.Background(), created, &kbclient.CreateOptions{}); err != nil {
+	if err := retry.OnError(retry.DefaultRetry, apierrors.IsAlreadyExists, func() error {
```

Comment:
I think it would make sense to check for generateName metadata before attempting to retry.

If object does not use generateName, retrying will just be redundant.

Reply:
I'm only adding this code in places where generateName is always set. If this were in a reusable func, then it would make sense to check that, but since the reusable func would end up just being a thin wrapper around `retry.OnError`, I'm not sure it's warranted here. Also, most of these calls involve interface objects so we'd have to attempt to cast to ObjectMeta, then look for GenerateName. I'm leaning towards thinking we don't need this since every time we call OnError it's right after explicitly setting generateName.

Reply:
cool.

Reply:
Upon further reflection, I think this is a good idea, actually -- I'll move the retry logic into a util func which either does retries or a simple one-time call, depending on GenerateName/Name values.